app-laptop/tpacpi-bat/files/tpacpi-bat.confd.0

```diff
@@ -3,6 +3,9 @@
 # use '/etc/init.d/tpacpi-bat info' at any time to check the thresholds
 # currently used by the driver
 
+# number of batteries you have
```

Comment:
If you like you could implement that, too. I mainly wanted to ensure that a user could pick the batteries he wants to set the threshold on, but allowing to leave one out.